### PR TITLE
COMPASS-3797 Update home plugin to use new connection model

### PR DESCRIPTION
**Pull request by alenakhineika**
The home plugin refactoring to recognize the new data structure, that is being returned from the new connection model.

### CODE REVIEW

**Comment by alenakhineika** [comment]
@durran, @aherlihy updated! could you pls check it when you will have time.

File: .travis.yml [hunk 1]
```diff
@@ -5,14 +5,13 @@ node_js:
...
   matrix:
-    - MONGODB_VERSION=4.0.x MONGODB_TOPOLOGY=standalone
+    - MONGODB_VERSION=stable MONGODB_TOPOLOGY=standalone
 addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
     packages:
       - libkrb5-dev
-      - libstdc++6
```
Comment (aherlihy) [inline .travis.yml]: Why was this removed?

File: package.json [hunk 2]
```diff
@@ -66,6 +67,7 @@
     "@mongodb-js/compass-import-export": "^1.3.0",
     "@mongodb-js/compass-indexes": "0.0.10",
     "@mongodb-js/compass-instance": "^1.2.0",
+    "@mongodb-js/compass-instance-header": "^0.2.2",
```
Comment (aherlihy) [inline package.json]: I think we should get rid of this, right? Since it is no longer being used. It should be removed from the renderer so that it's not required for it to run in standalone mode.

File: .travis.yml (hunk 1 above)
Comment (alenakhineika) [reply .travis.yml]: Nice catch! Tests on travis were failing and I was experimenting with configuration. Forgot to return this back.

File: package.json (hunk 2 above)
Comment (alenakhineika) [reply package.json]: Removed this plugin.

**Comment by alenakhineika** [comment]
@durran updated!
